src/ui/components/CreateIdentifier/components/IdentifierStage1BodyInit.tsx

```diff
@@ -23,86 +25,96 @@ const IdentifierStage1BodyInit = ({
   };
 
   return (
-    <ResponsivePageLayout
```

Comment:
No need to change all this. There is no delete button here.

![12092024153652](https://github.com/user-attachments/assets/de74a2f3-04c1-4495-a62a-86060df5577b)